lib/service/Controller/Swap/AXT.dart

```diff
@@ -8,6 +8,13 @@ class AXT extends Token with ERC20Behavior {
   static final String mumbaiAddress =
       "0x76d9a6e4cdefc840a47069b71824ad8ff4819e85";
```

Comment:
Would it useful to do something like 
static final String txAddress = (kReleaseMode) ? polygonAddress : mumbaiAddress

Reply:
I guess we would need to hear from @user2745. I was planning to switch the addresses when a user switches the network. Since testnet and mainnet have different chainID's. Or we can make it depend on release mode. Do we want to leave testnet working on the production version?

Reply:
I personally dont think that has any benefit to the user but I will let @user2745 chime in